suite-native/accounts/src/transactionCacheMiddleware.ts

```diff
@@ -0,0 +1,30 @@
+import { TransactionCacheEngine } from '@suite-common/transaction-cache-engine';
+import { createMiddlewareWithExtraDeps } from '@suite-common/redux-utils';
+import { accountsActions } from '@suite-common/wallet-core';
+
+export const prepareTransactionCacheMiddleware = createMiddlewareWithExtraDeps(
+    (action, { next }) => {
+        if (
+            accountsActions.updateAccount.match(action) ||
+            accountsActions.createAccount.match(action)
+        ) {
+            if (action.payload.networkType === 'ripple') {
+                TransactionCacheEngine.addAccount({
+                    coin: action.payload.symbol,
+                    descriptor: action.payload.descriptor,
+                });
+            }
+        }
```

Comment:
Isn't this duplication of `suite-native/accounts/src/useTransactionCache.ts`. Maybe I'm missing something 🤔

If new account gets created, it will trigger useEffect in the hook and call the same method, no?

Reply:
Good catch this is not neccessary anymore